lib/bootstrap_form/inputs/base.rb

```diff
@@ -22,6 +22,7 @@ def bootstrap_select_group(field_name)
           with_field_name = "#{field_name}_with_bootstrap"
           without_field_name = "#{field_name}_without_bootstrap"
           define_method(with_field_name) do |name, options={}, html_options={}|
+            html_options = html_options.reverse_merge(control_class: "form-select")
```

Comment:
I spent some time looking at this, refreshing my memory about how the code works. The `form_group_builder` method made more sense in earlier versions of Bootstrap, I think. I think your solution is good. I've created an issue #568 to look at whether `form_group_builder` is really serving a purpose now. But your PR is good.

Reply:
Good to hear. The metaprogramming involved made it a little harder to understand what was happening. 👍